app/models/time_reg.rb

```diff
@@ -53,6 +63,20 @@ def toggle_active
     where("date(date_worked) BETWEEN ? AND ?", start_date, end_date)
   }
 
+  protected
+
+  def only_one_active_time_reg
+    return unless active?
+
+    active_time_regs = self.user.time_regs.where.not(start_time: nil)
```

Comment:
if this PR gets merged we could maybe use the `all_active` scope https://github.com/rubynor/reap/pull/144/files

Reply:
:+1: